packages/base/src/3dview/mainview.tsx

```diff
@@ -1647,6 +1675,93 @@ export class MainView extends React.Component<IProps, IStates> {
     window.dispatchEvent(resizeEvent);
   }
 
+  private _updateSplit(enabled: boolean) {
+    if (enabled) {
+      if (!this._meshGroup) {
+        return;
+      }
+      this._renderer.setScissorTest(true);
+
+      this._sliderPos = (this._divRef.current?.clientWidth ?? 0) / 2;
+      this._sceneL = new THREE.Scene();
+      this._sceneL.background = SPLITVIEW_BACKGROUND_COLOR;
+      this._sceneL.add(new THREE.AmbientLight(0xffffff, 0.5)); // soft white light
+      const light = new THREE.HemisphereLight(0xffffff, 0x444444, 0.5);
+      this._sceneL.add(light);
+      this._sceneL.add(this._meshGroup.clone(true));
```

Comment:
No, the scene on the left has nothing to do with the OCC worker, it is just a snapshot of the current mesh at the time of checking out. The scene on the right is connected to the forked y-document and it works just like a normal 3D view.

This is why it isn't very easy to get back to the original document from a forked one since the forked y-doc in the frontend will propagate changes back to the original doc in the backend.

Reply:
Ok got it.

So the `sceneL` is not connected to the original document anymore, meaning if another collaborator changes the base document without making a suggestion, it is not reflected in `sceneL` **unless they refresh the page**. We may consider this as a bug? Happy to merge the PR as is anyway and we can iterate upon.